content-repo/extra-docs/articles/microsoft-auth-guide.md

```diff
@@ -242,4 +242,19 @@ After you a redirected to the next page, in the **Overview** tab you will find y
 | Microsoft Graph User | yes | yes - support both client secret and certificate | no | yes | yes |
 | Microsoft Management Activity API (O365 Azure Events) | yes | no | no | yes | yes |
 | Microsoft Teams | no | yes | no | yes | no |
-| Microsoft Teams Management | yes | yes | yes | no | yes |
\ No newline at end of file
+| Microsoft Teams Management | yes | yes | yes | no | yes |
+
+
+## Troubleshooting
+In case of errors in the authentication process - token revoked/expired or in case you generate new credentials, 
+you can use the `*!<integration command prefix>-auth-reset*` command in the War Rooms in order to rerun the authentication process,
+instead of recreating a new integration instance.
+Following verify the connectivity by clicking the Test button to test the instance.
+
+For example, for Microsoft Graph Mail Single User,
```

Comment:
```suggestion
For example, when using the "self-deployed Azure app for Microsoft Graph Mail Single User, in case of an expired/revoked token error:
```